.github/workflows/ci.yml

```diff
@@ -0,0 +1,31 @@
+name: ci
+on:
+  pull_request:
+  push:
+    branches:
+      - main
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out repository
+        uses: actions/checkout@v4
+
+      - name: Configure Python
+        uses: actions/setup-python@v5
+        with:
+          python-version: "3.8"
+
+      - name: Building Collection scripts
+        run: pip install -e .
+
+  test:
+    name: "test (${{ matrix.python-version }})"
+    strategy:
+      fail-fast: true
+      matrix:
+        python-version: ["3.8"]
+        # python-version: ["3.8", "3.9", "3.10", "3.11", "3.12"]
```

Comment:
Do you mean to test only 3.8?

Reply:
I was undecided but, in hindsight, I should have put all versions back in place. I've rectified this.